Makefile

```diff
@@ -81,7 +81,7 @@ tc: bin/phpunit
 INFECTION=vendor-bin/infection/vendor/bin/infection
 tm:			 ## Runs Infection
 tm:	$(TU_BOX_DEPS) $(INFECTION)
-	$(PHPNOGC) $(INFECTION) --only-covered
+	cd vendor-bin/infection && $(PHPNOGC) vendor/bin/infection --only-covered
```

Comment:
Are you sure about this change? IMO it should be reverted.
Also `infection.json.dist` should be moved to the root of the project.

Reply:
Yes, as there is same problem we had with php-scoper. Infection relies on package-versions and it could not be found as it is installed with the bin plugin in a different dir.
Either we switch to phars via phive or install it via the root composer version

Reply:
The problem with php-scoper was that the current working directory was not the root of the project scanned.
Your change here is different.

Reply:
I've upgrade infection to latest version in #510 based on your phpunit upgrade #509 it seems to be working as long as composer v1 is used. With composer v2 it might be broken.

Reply:
Seems to work :-)